### ISSUE TITLE
Downloader downloads empty files

### ISSUE THREAD

**Issue by Nhoque3786** [weespin/KhinsiderDownloader#17]
Hi! i tried to use your downloader, but seems like for some reason, it only create empty files with the names of the music files (used minecraft's soundtrack as test for screenshot)

is there any way to fix this? 
![image](https://github.com/weespin/KhinsiderDownloader/assets/43646247/72b90342-81dc-453a-840a-038442b73155)

**Comment by RazzeeX** [comment]
Yes, same here. I tried downloading several albums, it's just a bunch of 0 Kb files (either MP3 or FLAC).

**Comment by Myaael** [comment]
Did you let the whole download finish? In the screenshot you've got the 'Cancel' option available, that means the process is still ongoing.

Tried the album myself and it works perfectly fine. I had a problem with my connection dropping before and that caused a bunch of 0kb and half downloaded files.

**Comment by weespin** [comment]
![image](https://github.com/weespin/KhinsiderDownloader/assets/17180451/80273a26-4473-4a6e-91a5-3bf9c29a4885)
Cannot reproduce. 
Try to wait until all the files are downloaded.
Note: Look at the title bar, it shows the current progress

**Comment by RazzeeX** [comment]
My connection is normal, and I'm not the same person who initiated the thread.

![image](https://github.com/weespin/KhinsiderDownloader/assets/69221199/c5b1d399-23b7-4c13-8d27-8fb70bfb9706)

All MP3 files and cover arts are 0 Kb files.

![image](https://github.com/weespin/KhinsiderDownloader/assets/69221199/c68a37cc-ee71-47c2-82c9-db4ec78f9b1d)

**Comment by weespin** [comment]
Should be fixed in [last release](https://github.com/weespin/KhinsiderDownloader/releases). If it's still not working, feel free to reopen the issue
